runtime/caml/memory.h

```diff
@@ -46,7 +46,7 @@ CAMLextern void caml_free_dependent_memory (value v, mlsize_t bsz);
 CAMLextern void caml_modify (volatile value *, value);
 CAMLextern void caml_modify_local (value obj, intnat i, value val);
 CAMLextern void caml_initialize (volatile value *, value);
-CAMLextern int caml_atomic_cas_field (value, intnat, value, value);
```

Comment:
same here, I don't think we should rename this function.

Reply:
reverted